cmd/kpod/rm.go

```diff
@@ -39,20 +45,44 @@ func rmCmd(c *cli.Context) error {
 	defer runtime.Shutdown(false)
 
 	args := c.Args()
-	if len(args) == 0 {
+	if len(args) == 0 && !c.Bool("all") {
 		return errors.Errorf("specify one or more containers to remove")
 	}
 
-	for _, container := range args {
-		ctr, err := runtime.LookupContainer(container)
+	var delContainers []*libpod.Container
+	if c.Bool("all") {
+		delContainers, err = runtime.GetContainers()
+		if err != nil {
+			return errors.Wrapf(err, "unable to get container list")
+		}
+	} else {
+		for _, i := range args {
+			container, err := runtime.LookupContainer(i)
+			if err != nil {
+				fmt.Fprintln(os.Stderr, err)
```

Comment:
If you move lastError up next to delContainers and assign to it here, we can avoid a potential case where there was an error identifying containers to remove, but all the ones we did identify were removed successfully

Reply:
fixed